packages/server/lib/util/validation.js

```diff
@@ -104,18 +104,19 @@ const isValidBrowserList = (key, browsers) => {
 }
 
 const isValidRetriesConfig = (key, value) => {
-  const isNullOrNumber = isOneOf([_.isNumber, _.isNull])
-
-  if (
-    isNullOrNumber(value)
-    || (_.isEqual(_.keys(value), ['runMode', 'openMode']))
-      && isNullOrNumber(value.runMode)
-      && isNullOrNumber(value.openMode)
-  ) {
+  const optionalKeys = ['runMode', 'openMode']
+  const isValidRetryValue = (val) => _.isNull(val) || (_.isNumber(val) && val >= 0)
```

Comment:
Should we go ahead and validate that it's an integer? Make sure some joker doesn't try `retries: 2.3`?

Reply:
sure why not